src/javascript/app_2/Stores/Modules/Trading/Helpers/proposal.js

```diff
@@ -74,3 +74,34 @@ const createProposalRequestForContract = (store, type_of_contract) => {
         ),
     };
 };
+
+export const getProposalParametersName = (proposal_requests, alternatives, removable_properties) => {
+    const is_digit = Object.keys(proposal_requests)
```

Comment:
Can we call [`isDigitContract()`](https://github.com/binary-com/binary-static/blob/new-app/src/javascript/app_2/Stores/Modules/Contract/Helpers/digits.js#L1) instead?

Reply:
@binary-hamedanchi 
IMO it's not appropriate to use it since `proposal.js` and `digits.js` are in different modules unless we move the `digit.js` in a more common directory.